web-app/src/main/resources/templates/fragments/header.html

```diff
@@ -0,0 +1,8 @@
+<th:block th:fragment="header">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<title>Start</title>
```

Comment:
Tytuł będzie się taki sam dla każdej strony. 
Dałoby radę wyciągnąć tag <title> i wprowadzać go osobno dla każdego html'a?

Reply:
Póki co można to przepuścić ale trzeba by było zrobić z tego taska żeby nam nie umknęło.

Reply:
Dodałem zmienną tytuł